node/chainSimulator/components/cryptoComponents.go

```diff
@@ -21,6 +21,7 @@ type ArgsCryptoComponentsHolder struct {
 	Preferences             config.Preferences
 	CoreComponentsHolder    factory.CoreComponentsHolder
 	ValidatorKeyPemFileName string
+	SkKeyIndex              int
```

Comment:
SkKexIndex means SecretKeyKeyIndex. might remove the `Key` from the variable's name

Reply:
refactored.